CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing to Dials
+
+We're happy to consider contributions from outside sources; whether in the form
+of code, tickets, documentation or even just typo corrections! We're aware that
+Dials can at times appear to have relatively indiosyncratic code, and some of
```

Comment:
This comes across too negative in my opinion. Rather than "idiosyncratic" and "obscure/niche" I would just say that "DIALS addresses a wide range of use cases - so, if in the event..."